include/SMaterialLayer.h

```diff
@@ -43,14 +43,38 @@ namespace video
 			"texture_clamp_mirror_clamp_to_edge",
 			"texture_clamp_mirror_clamp_to_border", 0};
 
+
+	//! Texture minification filter.
+	/** Used when scaling textures down. */
+	enum E_TEXTURE_MIN_FILTER {
+		//! Nearest-neighbor interpolation.
+		ETMINF_NEAREST = 0,
+		//! Linear interpolation.
+		ETMINF_BILINEAR,
+		//! Linear interpolation across mipmaps.
+		/** Is equivalent to ETMINF_BILINEAR if mipmaps are disabled.
+		Only available as a minification filter since mipmaps are only used
+		when scaling down. */
+		ETMINF_TRILINEAR,
+	};
+
+	//! Texture magnification filter.
+	/** Used when scaling textures up. */
+	enum E_TEXTURE_MAG_FILTER {
+		//! Nearest-neighbor interpolation.
+		ETMAGF_NEAREST = 0,
+		//! Linear interpolation.
+		ETMAGF_BILINEAR,
+	};

```

Comment:
Maybe use actual OpenGL tokens?

Reply:
I don't think we can use the actual `GL_…` enum values if we want proper type checking, but I'll at least give them the same names as the OpenGL enum values.

Reply:
The `ETMINF_…` and `ETMAGF_…` enum values now have the same names as the `GL_…` enum values.